app/models/form/sales/pages/housing_benefits.rb

```diff
@@ -14,7 +14,7 @@ def depends_on
     if @joint_purchase
       [{ "joint_purchase?" => true }]
     else
-      [{ "not_joint_purchase?" => true }]
+      [{ "not_joint_purchase?" => true }, { "jointpur" => nil }]
```

Comment:
Joint purchase is routed to conditionally, so might not always be answered, added this here just to be sure we have default question

Reply:
Hm, good point - still displaying this question for nil jointpurs makes sense to me - will this make any previously complete logs become in progress if they hadn't satisfied the condition for jointpur to be routed to?